src/Sulu/Bundle/ActivityBundle/Resources/config/services.xml

```diff
@@ -74,5 +74,16 @@
             <tag name="sulu.admin"/>
             <tag name="sulu.context" context="admin"/>
         </service>
+
+        <service
+            id="sulu_activity.list_metadata_provider"
+            class="Sulu\Bundle\ActivityBundle\Infrastructure\Sulu\Metadata\ActivityListMetadataProvider"
+            decorates="sulu_admin.list_metadata_provider"
+        >
+            <argument type="service" id=".inner"/>
```

Comment:
think we need to use `sulu_activity.list_metadata_provider.inner` for symfony 4: https://symfony.com/blog/new-in-symfony-5-1-simpler-service-decoration